x/staking/legacy/v0_36/migrate.go

```diff
@@ -26,9 +29,14 @@ func migrateValidators(oldValidators v034staking.Validators) Validators {
 	validators := make(Validators, len(oldValidators))
 
 	for i, val := range oldValidators {
+		consPubKey, err := sdk.Bech32ifyConsPub(val.ConsPubKey)
```

Comment:
For bervity and to keep in line with how migration types are typically defined, we should rather copy over the JSON serialization logic. i.e. we need to copy `MarshalJSON` and `UnmarshalJSON`

https://github.com/cosmos/cosmos-sdk/blob/v0.36.0/x/staking/types/validator.go#L180